README.md

```diff
@@ -37,6 +37,10 @@ Add your extension to the index to make it available in these CLI commands:
 
 ### How to generate sha256digest for an index.json entry?
 
+The `azdev extension update-index` command will calculate the SHA256 digest for you and add your entry to the index.json file. For more information visit https://github.com/Azure/azure-cli-dev-tools.
+
```

Comment:
It might be good to make this message more consistent with the message for the metadata below.

Reply:
I updated it to only refence that it calculates the SHA256 digest. So it's actually less similar but I think it makes more sense.